### ISSUE TITLE
One Too Many Allowed Attempts

### ISSUE THREAD

**Issue by jefferysmith** [grygoriy/bruteforcedefender#4]
Maybe it is just a wording issue, but the plugin actually allows allowedNumberOfAttempts +1 attempts before locking out the user.
For example, for  `allowedNumberOfAttempts = 3` the user is not locked until after the 4th failed login attempt. I would have expected them to be locked out after the 3rd attempt.

**Comment by grygoriy** [comment]
yes your are right, this is bad explanation.
Currently meaning should be like 3 attempts is allowed, How would you
rephrase it to make more clear?

thank you for mailing this
On 24 October 2012 00:57, Jeff Smith notifications@github.com wrote:

> Maybe it is just a wording issue, but the plugin actually allows
> allowedNumberOfAttempts +1 attempts before locking out the user.
> For example, for allowedNumberOfAttempts = 3 the user is not locked until
> after the 4th failed login attempt. I would have expected them to be locked
> out after the 3rd attempt.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/grygoriy/bruteforcedefender/issues/4.

## 

Grygoriy Mykhalyunyo

skype:Grygoriy.Mykhalyunyo
tel: +38 097 544 72 53
jabber: grishaster@gmail.com

**Comment by jefferysmith** [comment]
_(Sorry, accidentally clicked on the close button while adding a comment)._

It makes the most sense to me for this setting mean that the account will become locked after that number of failed attempts - which means that the existing logic needs to change to lock the account one attempt earlier than it currently does. So if I set the value to 3, I would expect that the account should be locked after the third failed attempt.
As for the name, something like 'maxLoginAttempts' might be a bit clearer to read, but that is more of a subjective thing.

And thank-you for creating the plugin. It is quite useful.

**Comment by grygoriy** [comment]
thank you you response and submiting this!
I will change this behaviour
